src/media/js/nav.js

```diff
@@ -152,4 +152,61 @@ define('nav', ['core/log', 'core/navigation', 'core/views', 'core/z'],
     .on('click', '.settings-menu-desktop a', function() {
         z.page.trigger('clearsettings');
     });
+
+    // Firefox OS 1.1 compatibility 
+    if(navigator.userAgent.match(/Mobile/) && navigator.userAgent.match(/Firefox\/18\.1/)) {
+        // we must recreate all the overlay events and animations with jQuery
+        var overlays11 = function() {
+            // loop until all the elements are ready
+            if($(".above,.below,.header-categories-btn,.global-nav-link,#navigation,.cat-menu-all a").length) {
+                // hide it first
+                $(".above").slideUp();
+                $(".below").css({ // below is more complicated, since it comes from below
+                    "top": ($(document).height())+"px",
+                    "display": "none"
+                });
+
+                // bug fix - removes "Categories" header link
+                $(".cat-menu-all a").attr('href', 'javascript:void(0);');
+
+                // sets width to 100% of the page
+                $(".above,.below,.global-nav-menu").width($(window).width()+"px");
+                // do it again if the user toggles portrait/landscape mode
+                $(window).resize(function() {
+                    $(".above,.below,.global-nav-menu").width($(window).width()+"px");
+                });
+
+                // open events
+                $(".header-categories-btn").click(function() {
+                    $(".above").slideDown();
```

Comment:
we should save these are `var $above = $('.above')` and do the same for `$below`